dotnetv3/dynamodb/scenarios/PartiQL_Batch_Scenario/PartiQL_Batch_Scenario/PartiQLBatch.cs

```diff
@@ -9,8 +9,8 @@
 
 // Separator for the console display.
 var SepBar = new string('-', 80);
-const string tableName = "movie_table";
-const string movieFileName = "moviedata.json";
+const string tableName = "movie_table6";
+const string movieFileName = @"..\..\..\..\..\..\..\..\resources\sample_files\movies.json";
```

Comment:
Same comment about movies.json. If we point them to the file in resources, why ask them to downdown it from DEV guide. Seems unnecessary.
.

Reply:
removed the comment.